### PR TITLE
use plaid-go beta clib

### PR DESCRIPTION
**Pull request by otherchen**
* updates the go quickstart to use the beta plaid-go client library

### CODE REVIEW

**Review by heangpov16 — APPROVED** [review]

**Comment by heangpov16** [comment]
support heangpov16

File: go/server.go [hunk 1]
```diff
@@ -156,187 +156,251 @@ func getAccessToken(c *gin.Context) {
...
+	).Execute()
+
 	if err != nil {
 		renderError(c, err)
 		return
 	}
 
 	c.JSON(http.StatusOK, gin.H{
-		"payment": response.Payment,
+		"payment": paymentGetResp,
```
Comment (stephenjayakar) [inline go/server.go:341]: is this right? seems that this is a different schema

File: go/server.go [hunk 2]
```diff
@@ -349,42 +413,63 @@ func createLinkToken(c *gin.Context) {
...
-			error:     err.Error(),
-		}
+		return "", err
 	}
-	return resp.LinkToken, nil
+
+	return linkTokenCreateResp.GetLinkToken(), nil
 }
 
 func assets(c *gin.Context) {
```
Comment (stephenjayakar) [inline go/server.go:475]: so this isn't exactly true anymore i think. we should probably modify the error message we return if we don't want to support this.

**Review by stephenjayakar — APPROVED** [review]
thanks for improving patterns as you saw them :). and thank you for updating. just one comment

**Comment by otherchen** [comment]
@stephenjayakar should be ready for another look! I added support for assets for the go clib.

**Pull request merged**